### PR TITLE
Adds embeddings filter option to return scores in state

### PR DESCRIPTION
**Pull request by jacoblee93**
CC @baskaryan @assafelovic

### CODE REVIEW

**Comment by assafelovic** [comment]
This is great! Looking forward to using after merge. @baskaryan

File: libs/langchain/langchain/retrievers/document_compressors/embeddings_filter.py [hunk 1]
```diff
@@ -33,6 +33,8 @@ class EmbeddingsFilter(BaseDocumentCompressor):
     """Threshold for determining when two documents are similar enough
     to be considered redundant. Defaults to None, must be specified if `k` is set
     to None."""
+    return_similarity_scores = False
```
Comment (baskaryan) [inline libs/langchain/langchain/retrievers/document_compressors/embeddings_filter.py:36]: probably don't even need to make this configurable, could just always return score in state

**Comment by jacoblee93** [comment]
Updated

**Comment by assafelovic** [comment]
looking good! lmk when merged @baskaryan

**Pull request merged**